### PR TITLE
Adjust calendar spacing

### PR DESCRIPTION
**Pull request by JoomFX**
Adjust calendar spacing to accommodate both multiview and classic calendar designs.

### CODE REVIEW

File: packages/bootstrap/scss/calendar/_variables.scss [hunk 1]
```diff
@@ -4,16 +4,16 @@ $calendar-font-family: $font-family !default;
 $calendar-font-size: $font-size !default;
 $calendar-line-height: $line-height !default;
 
-$calendar-header-padding-x: $padding-x !default;
-$calendar-header-padding-y: $padding-y !default;
-$calendar-header-padding-x-sm: ($calendar-header-padding-y / 2) !default;
+$calendar-header-padding-x: ($padding-x / 4) !default;
```
Comment (joneff) [inline packages/bootstrap/scss/calendar/_variables.scss:7]: use `map-get( $spacing, ... )` to achieve the same result.

File: packages/bootstrap/scss/calendar/_variables.scss [hunk 2]
```diff
@@ -34,6 +34,7 @@ $calendar-navigation-item-height: 2em !default;
 
 $calendar-view-width: 7 * $calendar-cell-size !default;
 $calendar-view-height: calc( #{$calendar-cell-size * 8} + #{$calendar-header-height} ) !default;
+$calendar-view-gap: 16px !default;
```
Comment (joneff) [inline packages/bootstrap/scss/calendar/_variables.scss:37]: same here

File: packages/bootstrap/scss/calendar/_variables.scss [hunk 3]
```diff
@@ -63,8 +64,7 @@ $calendar-navigation-bg: $calendar-header-bg !default;
 $calendar-navigation-text: $calendar-header-text !default;
 $calendar-navigation-border: $calendar-header-border !default;
 
-$calendar-today-nav-margin-x: 8px !default;
-$calendar-today-nav-margin-y: 0 !default;
+$calendar-nav-gap: 4px !default;
```
Comment (joneff) [inline packages/bootstrap/scss/calendar/_variables.scss:67]: same here

File: packages/default/scss/calendar/_layout.scss [hunk 4]
```diff
@@ -131,10 +131,7 @@
                 flex-direction: row;
```
Comment (joneff) [inline packages/default/scss/calendar/_layout.scss:131]: combine `flex-direction` and `flex-wrap` into `flex-flow: row nowrap`

File: packages/default/scss/calendar/_layout.scss [hunk 5]
```diff
@@ -153,7 +150,7 @@
         }
         .k-nav-today,
         .k-footer > .k-state-disabled {
-            padding: $calendar-header-padding-y $calendar-header-padding-x;
+            padding: ($calendar-header-padding-y * 2) ($calendar-header-padding-x * 4);
```
Comment (joneff) [inline packages/default/scss/calendar/_layout.scss:153]: perhaps `$calendar-footer` variables?

File: packages/default/scss/calendar/_layout.scss [hunk 6]
```diff
@@ -338,6 +338,7 @@
             .k-calendar-header {
                 margin-left: -$calendar-content-padding-x;
                 margin-right: -$calendar-content-padding-x;
+                padding: ($calendar-header-padding-y * 2) ($calendar-header-padding-x * 4);
```
Comment (joneff) [inline packages/default/scss/calendar/_layout.scss:341]: why is the variable called `$calendar-header-padding-` if we need to multiply it to get the values in calendar header?

**Review by joneff — CHANGES_REQUESTED** [review]

File: packages/default/scss/calendar/_layout.scss (hunk 6 above)
Comment (JoomFX) [reply packages/default/scss/calendar/_layout.scss:341]: The idea is to use the `$calendar-header-padding-` variable (without multiplying it) for all other Calendar Headers and only multiply it for the Infinite Calendar Header.
Should I introduce a new variables instead, for example `$calendar-infinite-header-padding-`?

**Review by joneff — APPROVED** [review]

**Review by PreslavKozovski — APPROVED** [review]

**Pull request merged**